docs/installing-solidity.rst

```diff
@@ -197,7 +197,13 @@ Building Solidity is quite similar on Linux, macOS and other Unices:
     cd build
     cmake .. && make
 
-And even on Windows:
+or even easier:
+
+.. code:: bash
+
+    ./scripts/build.sh --Release #specifies the build type
```

Comment:
fair enough. This is all in the previous PR FYI...but there have been some problems...it seems I may have been too quick to be super confident that the rpc tests would pass like that.